src/cmd/check.go

```diff
@@ -54,22 +56,74 @@ var listCheckCmd = &cobra.Command{
 var checkCreateCmd = &cobra.Command{
 	Use:   "check",
 	Short: "Create a rubric check",
-	Long: `Create a rubric check
-
-Examples:
-
-	opslevel create check -f my_cec.yaml
-`,
+	Long:  `Create a rubric check`,
+	Example: `
+cat << EOF | opslevel create check -f -
+Version: "1"
+kind: "repo_grep"
+spec:
+  name: "new repo grep check"
+  enabled: false
+  category: "misc"
+  level: "silver"
+  notes: "some notes"
+  directorySearch: false
+  filePaths:
+  - "Taskfile.yml"
+  fileContentsPredicate:
+    type: "exists"
+    value: ""
+EOF`,
 	Run: func(cmd *cobra.Command, args []string) {
-		usePrompts := !hasStdin()
-		input, err := readCheckCreateInput()
+		input, err := readCheckInput()
 		cobra.CheckErr(err)
+		usePrompts := true
+		if cmd.Flag("noninteractive").Changed {
```

Comment:
I added a `--noninteractive` flag for when users want to skip using the prompt UI, this is useful for automations. `hasStdin()` isn't a reliable function, see https://github.com/OpsLevel/cli/pull/147

Reply:
This change could break things if the user upgrades their CLI version without realizing this flag was added and the old check was removed for being unreliable.

Reply:
Lets make sure we add a changie entry for this calling out this breaking change and what users must do to get the old behavior back.